api/guides/delegate_descriptor_ids.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Delegate Descriptor IDs
+---
+
+# Understanding Delegate Descriptor IDs 
+
+When a data element, condition, event, or action is associated with an extension package, that resource is considered a "delegate" of that package. Each delegate within an extension package is given a unique delegate descriptor ID. The delegate descriptor ID for a particular resource allows Adobe Experience Platform Launch to determine what kind of resource it is and which extension package it belongs to.
```

Comment:
I think it would be a good idea to start at an even higher level.  We should explain that anything Launch users can add to libraries and deploy to their sites is provided by extensions.  The capabilities of each extension are defined by the extension developer.  Functionality that developers put into extensions are packaged as "delegates" and they all have descriptor IDs.  

The only ones that the user will need to specify are Data Elements and Rule Components.  Rule Components come in three types: events, conditions, and actions.